modules/javafx.controls/src/test/java/test/javafx/scene/control/skin/ScrollPaneSkinTest.java

```diff
@@ -28,6 +28,15 @@
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
 
+import java.lang.ref.WeakReference;
+import java.util.ArrayList;
+
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
```

Comment:
These got moved from line 51(without this PR), can restore this change.

Reply:
changed formatter rules for import ordering: java,javax,javafx,org